### DISCUSSION TITLE
How to access the SD card once you've set up as a USB gadget?

### DISCUSSION THREAD

**Discussion by Alan-Angel** [probonopd/MiniDexed#663] [Q&A] [answered]
That's it really, even if I remove the SD card and plug it into my pc I can't see the card contents, so I can't update them without formatting through 3rd party disk imaging software and re-writing the whole thing?

**Comment by diyelectromusic** [comment, marked as answer]
I'm not really sure what issue you're describing I'm afraid.

The SD cards should be FAT formatted and MiniDexed files are just copied on - no imaging should be required.  How the cards are accessed is independent of USB gadget mode.

But note that MiniDexed does not provide "live" access to the SD card when plugged into a USB host - the USB connection is MIDI only.

Are you able to elaborate a little more what you're seeing and what you're doing in response?

**Reply by Alan-Angel** [reply]
Hi, I've tried again today and it's working. For most of yesterday whenever I put in an SD card that had minidexed on it and I'd set it to usbgadget=1, the sd card would no long appear up in explorer when I used a memory card reader (out of the pi). I mentioned the imaging software because the only way to recognise and reformat the card was by using that. There was no other way to see it. Anyway it seems to have fixed itself now. Thanks.
